test/system/inspect-provenance.test.ts

```diff
@@ -1,8 +1,8 @@
 /* eslint-disable @typescript-eslint/no-non-null-assertion */
 import { test } from 'tap';
-import { chdirWorkspaces, activateVirtualenv } from './test-utils';
+import { chdirWorkspaces, activateVirtualenv } from '../test-utils';
 
-import pluginImpl = require('../lib');
+import pluginImpl = require('../../lib');
```

Comment:
why require?

Reply:
just moved to test/system - requires refactor to jest in separate PR

Reply:
I think `import { inspect } from '../../lib';` and replacing `pluginImpl.inspect` with `inspect` would do the job